test/main.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "pybind11/numpy.h"
 
+#include "xtensor/xarray.hpp"
```

Comment:
Is this required?

Reply:
Yes.  On OSX if commenting it out, I got:
```
In file included from /Users/yungyuc/hack/code/xtensor-python/test/main.cpp:17:
In file included from /Users/yungyuc/hack/code/xtensor-python/include/xtensor-python/pyarray.hpp:17:
/Users/yungyuc/hack/usr/opt36_180914/include/xtensor/xiterator.hpp:406:25: error: no member named 'mpl' in namespace 'xtl'
            return xtl::mpl::static_if<has_storage_iterator<C>::value>([&](auto self)
                   ~~~~~^
```

Reply:
Hum, this seems to be something that was fixed about 2 months ago in https://github.com/QuantStack/xtensor/commit/8a6169219080a9beb7f8bd951d50dca73ce55e15.patch.

Reply:
You are right.  I was testing against 18a26c81e1335dccb513866684e5e6fbaeffcf25 (Fri Sep 14 15:55:58 2018 +0200).  After updating xtensor to the latest master (d61b71e75c1f555c6f014a9974e9f81045ce3bc6) I don't need this line anymore.  I'll remove it.

Reply:
@yungyuc excellent!

(By the way, is there a specific reason why you are running on specific commits and not version tags?)

Reply:
@SylvainCorlay It's only because of my careless practice.  My time on my side project is limited so I am developing against master of primary dependencies (pybind11, xtensor, xtensor-python, etc.).  It's fine but sometimes (like this time) I was distracted for weeks and forgot to update the dependencies.

Sorry for my silly mistake.  I've add a new change reverting this line and other two spots of unnecessary change.